packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -121,11 +124,34 @@ export class AtomicResultList implements InitializableComponent {
     }
   }
 
+  private renderPlaceholder() {
+    const results = [];
+    for (let i = 0; i < this.resultPerPageState.numberOfResults; i++) {
+      results.push(
+        <div class="flex pl-5 pt-5 mb-5">
+          <div class="w-14 h-14 bg-divider mr-8"></div>
+          <div class="flex-grow">
+            <div>
+              <div class="flex justify-between mb-5">
+                <div class="h-3 bg-divider w-1/2"></div>
+                <div class="h-2 bg-divider w-1/6"></div>
+              </div>
+              <div class="h-2 bg-divider w-4/6 mb-3"></div>
+              <div class="h-2 bg-divider w-5/6 mb-3"></div>
+              <div class="h-2 bg-divider w-5/12"></div>
+            </div>
+          </div>
+        </div>
+      );
```

Comment:
There is not list/card layouts, adding layout type will be a future feature, placeholder can reflect this. To know the height of an average result, it would have to be set by some kind of prop I assume. We can improve this later as well.